en/concept/flight_modes.md

```diff
@@ -14,23 +14,23 @@ The sections below provide an overview of the modes, followed by a [flight mode
 "Manual" modes are those where the user has direct control over the vehicle via the RC control (or joystick). Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode. For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
 
 * **Fixed wing aircraft/ rovers / boats:** 
-  * **MANUAL:** The pilot's control inputs are passed directly to the output mixer.
-  * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command.
+  * **MANUAL:** The pilot's control inputs or RC commands are passed directly to the output mixer.
```

Comment:
Not sure about this - the control inputs are RC commands. What is it you're trying to capture here?

Reply:
When I was first reading the dev guide without reading the user guide I got confused thinking "pilot" meant the "autopilot" and the "control inputs" were the commands computed by the autopilot. Maybe this confusion arose because I have always learned in my control systems lectures that "control inputs" is the term commonly used for commands computed by the controller for correction of errors, whereas in the manual mode (correct me if I am wrong) the user just sends the RC commands as "references"  and an open loop control is involved. 
Maybe this confusion is personal and due to not reading the user guide before. 
But if you think others like me can also get confused then it will be nice to add "(RC commands)" in brackets next to the "pilot's control inputs" the first time it is used for further clarification of where the "control inputs" are actually coming from.

Reply:
If you can make this assumption someone else might too. How about "The user's input from the RC transmitter ..."?

Reply:
@barzanisar Even better, how about "The raw inputs from the RC receiver"

Reply:
Since everywhere in the document it is written "pilot's inputs" so rather than changing everywhere, it will be easier to mention in brackets the first time pilot's inputs are used with (raw commands from the RC). What do you think?